dev/test/util/helpers.ts

```diff
@@ -334,6 +334,9 @@ export const postConverter = {
   toFirestore(post: Post): DocumentData {
     return {title: post.title, author: post.author};
   },
+  toFirestoreFromMerge(post: Partial<Post>): DocumentData {
+    return Object.assign({}, post);
```

Comment:
This is so 2019. All the cool kids use `{... post}`. :)

Reply:
However futile it might be, I appreciate you trying to up my cool factor